src/Zikula/CoreInstallerBundle/Helper/ParameterHelper.php

```diff
@@ -148,12 +148,27 @@ public function finalizeParameters(bool $configureRequestContext = true): bool
 
         if (isset($params['upgrading'])) {
             $params['zikula_asset_manager.combine'] = false;
-            $startController = $this->variableApi->getSystemVar('startController');
-            [$moduleName] = explode(':', $startController);
-            if (!$this->kernel->isBundle($moduleName)) {
-                // set the 'start' page information to empty to avoid missing module errors.
-                $this->variableApi->set(VariableApi::CONFIG, 'startController', '');
-                $this->variableApi->set(VariableApi::CONFIG, 'startargs', '');
+
+            // unset start page information if needed to avoid missing module errors
```

Comment:
"if needed"... not sure this is a good idea. easier to simply unset it all and allow the site admin to reset manually.

Reply:
sounds too much to do this on *every* update... maybe we could detect if it is a major upgrade?

Reply:
disagree. It isn't _that_ much to do and is far safer.

Reply:
changed like you wish